docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,560 @@
+  as a template argument. It needs to be constructed from any other executor.
+  Please note, that the function signatures that can be scheduled using
+  ``then_execute``, ``bulk_sync_execute``, ``bulk_async_execute`` and
+  ``bulk_then_execute`` are slightly different (See the comment in
+  :hpx-pr:`4514` for more details).
+* The underlying executor of ``block_executor`` has been updated to a newer one.
+* We have added a parameter to ``auto_chunk_size`` to control the amount of
+  iterations to measure.
+* We have added performance counters type information (aggregating,
+  monotonically increasing, average count, average timer, etc.).
+* The tasks are now re-scheduled on the same worker thread they were suspended
+  on to avoid cache misses from moving from one thread to the other. This
+  behavior doesn't prevent the task from being stolen.
+* We have added a new configuration option ``hpx.exception_verbosity`` to allow
+  to control the level of verbosity of the exceptions (3 levels available).
+* ``broadcast_to``, ``broadcast_from``, ``scatter_to`` and ``scatter_from`` have
+  been added to the collectives, modernization of ``gather_here`` and
+  ``gather_there`` with futures taken by rvalue references. See the breaking
+  change on ``all_to_all`` in the next section. All the collectives do not need
+  supporting macros anymore (e.g. specifying the data types used for a
+  collective operation using ``HPX_REGISTER_ALLGATHER`` and similar are not
+  needed anymore).
+* A new API function to get the number of cores which are idle
+  (``hpx::get_idle_core_count``) and another one returning a bitmask
+  representing the currently idle cores (``get_idle_core_mask``).
+* We have added an experimental option to only enable the local runtime, you can
+  disable the distributed runtime with ``HPX_WITH_DISTRIBUTED_RUNTIME=OFF``. You
+  can also enable the local runtime by using the ``--hpx:local`` runtime option.
+* We fixed task annotations for actions.
+* The alias ``hpx::promise`` to ``hpx::lcos::promise`` is now deprecated. You
+  can use ``hpx::lcos::promise`` directly instead. ``hpx::promise`` will refer
+  to the local-only promise in the future.
+* We have added a ``prepare_checkpoint`` API function which calculates the
+  amount of necessary buffer space for a particular set of argumnets
+  checkpointed.
+* We have changed the |cmake| targets exposed to the user, it now includes
+  ``HPX::hpx``, ``HPX::wrap_main`` (``int main`` as the first |hpx| thread of the
+  application, see :ref:`starting HPX runtime <_starting_hpx>`),
+  ``HPX::plugin``, ``HPX::component``.
+* A new example is demonstrating how to create and use a wrapping executor
+  (``quickstart/executor_with_thread_hooks.cpp``)
+* A new example is demonstrating how to disable thread stealing during the
+  execution of parallel algorithms
+  (``quickstart/disable_thread_stealing_executor.cpp``)
+* Many bug fixes.
+
```

Comment:
There is one more thing added: all executor parameter hooks can now be exposed through the executor itself. This will allow to deprecate the `.with()` functionality on execution policies in the future. This is also a first step towards simplifying our executor APIs in preparation for the upcoming C++23 executors (senders/receivers).

Reply:
I didn't bother to rephrase as your name is on the copyright and my english is pretty bad, I hope you don't mind ;)